### PR TITLE
BETA: Register ali-samir.is-a.dev

### PR DESCRIPTION
**Pull request by TheThreeL**
Added `ali-samir.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/the-l.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheThreeL",
+        "email": "alisamir3384408@gmail.com"
+    },
+    "record": {
+        "TXT": "hi"
```
Comment (Stef-00012) [inline domains/the-l.json:7]: TXT records alone do nothing

File: domains/ali-samir.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheThreeL",
+        "email": "alisamir3384408@gmail.com"
+    },
+    "record": {
+        "CNAME": "2fa83007-d49e-4418-83a2-536205c253aa-00-tpnws5m92s2x.worf.replit.dev"
```
Comment (Stef-00012) [inline domains/ali-samir.json:7]: I guess you turned off your repl, replit says
"The Repl is currently not running."

File: domains/topeck.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheThreeL",
+        "email": "alisamir3384408@gmail.com"
+    },
+    "record": {
+        "CNAME": "topeck"
```
Comment (Stef-00012) [inline domains/topeck.json:7]: Invalid CNAME record

**Review by Stef-00012 — CHANGES_REQUESTED** [review]

File: domains/ali-samir.json (hunk 2 above)
Comment (TheThreeL) [reply domains/ali-samir.json:7]: Should the repil be always open?
If yes, try opening it again

**Pull request closed without merging**